src/oadp_resources/velero/backup.py

```diff
@@ -131,3 +135,11 @@ def wait_for_deleting(self, wait_timeout=200, sleep=1):
             sleep=sleep,
             wait_timeout=wait_timeout
         )
+
+    def wait_for_done(self, wait_timeout=200, sleep=1):
+        return wait_for(
+            condition_function=self.done,
+            description=f"{self.kind} backup to done, {self.name}",
```

Comment:
backup to be done Or the Backup process to finish.
wdyt?

Reply:
Finish, dine, is the same... nope?